lib/Styles/Gtk/Separator.scss

```diff
@@ -0,0 +1,11 @@
+$separator-color: rgba(black, 0.15);
+@if $color-scheme == "dark" {
+    $separator-color: rgba(black, 0.35);
+}
+
+separator {
```

Comment:
If you're specifically wanting to style pane seps and not all seps (in menus and all else) probably should create a Paned.scss and do `paned.vertical > separator`